lib/runtime/invalid_transaction_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package runtime
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/transaction"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInvalidTransactionErrors(t *testing.T) {
+	testCases := []struct {
+		name        string
+		test        []byte
+		expErr      error
+		expValidity *transaction.Validity
+	}{
+		{
+			name:   "ancient birth block",
+			test:   []byte{1, 0, 5},
+			expErr: errAncientBirthBlock,
+		},
+	}
+
+	for _, c := range testCases {
+		t.Run(c.name, func(t *testing.T) {
+			validity, err := UnmarshalTransactionValidity(c.test)
```

Comment:
nit run subtests in parallel.. although there is only 1 but still a good habit!

```suggestion
	for _, c := range testCases {
		c := c
		t.Run(c.name, func(t *testing.T) {
			t.Parallel()
			validity, err := UnmarshalTransactionValidity(c.test)
```